strict_encoding/derive/Cargo.toml

```diff
@@ -18,7 +18,7 @@ proc-macro = true
 [dependencies]
 syn = "1"
 proc-macro2 = "1"
-encoding_derive_helpers = { version = "0.9.0", path = "../derive_helpers" }
+encoding_derive_helpers = { version = "0.8.1", path = "../derive_helpers" }
```

Comment:
Be aware, a similar downgrade happened here. Was this one intentional?

Reply:
From what I see both this crate and `encoding_derive` were never released to `v0.9`. Probably I decided that since they do not have any changes, no point in doing a new release. I released `v0.9` after your comment above, but I think here we can skip that.

Reply:
Alright, that makes sense.